.github/workflows/update-contributors.yml

```diff
@@ -0,0 +1,40 @@
+name: Update Contributors
+
+on:
+  workflow_dispatch:
+  schedule:
+    # run at midnight every day
+    - cron: '0 0 * * *'
```

Comment:
I am afraid that updating the dataset every day is too frequent. After all, every version of the JSON files has to be recorded in the Git history. To contain the size of the Git history, I suggest updating the dataset once a week. For example, 4am in the morning on every Monday.